### PR TITLE
Add network URL to the PLN plugin settings page.

### PR DESCRIPTION
**Pull request by ubermichael**
The network URL defaults to a value defined in the code, and can be overridden in on defined in config.inc.php. Journal managers can also set a value for the network URL.

Closes pkp/pkp-lib#586

### CODE REVIEW

File: plugins/generic/pln/classes/form/PLNSettingsForm.inc.php [hunk 1]
```diff
@@ -82,22 +90,77 @@ function display() {
...
 		$templateMgr =& TemplateManager::getManager();
 		$templateMgr->assign('hasIssn', $hasIssn);
+		$templateMgr->assign('journal_uuid', $this->_plugin->getSetting($this->_journalId, 'journal_uuid'));
 		$templateMgr->assign('terms_of_use', unserialize($this->_plugin->getSetting($this->_journalId, 'terms_of_use')));
 		$templateMgr->assign('terms_of_use_agreement', $this->getData('terms_of_use_agreement'));
 		parent::display();
-	}  
+	}
+
+	function _networkChanged() {
```
Comment (asmecher) [inline plugins/generic/pln/classes/form/PLNSettingsForm.inc.php]: This function needs documentation.

File: plugins/generic/pln/classes/form/PLNSettingsForm.inc.php [hunk 2]
```diff
@@ -82,22 +90,77 @@ function display() {
...
+		$this->_plugin->getServiceDocument($this->_journalId);
+	}
+
+	/**
+	 * Custom network URL validation function.
+	 *
+	 * The PLN staging server must be run at the root of a domain name, and must
+	 * be http or https. An optional port may be specified.
+	 *
+	 * Sets the
```
Comment (asmecher) [inline plugins/generic/pln/classes/form/PLNSettingsForm.inc.php]: Sets the ...

**Comment by asmecher** [comment]
Just one comment, but this also needs rebasing.

**Comment by ubermichael** [comment]
Fixed and rebased.

**Comment by asmecher** [comment]
Sorry, it looks like merging #599 means a rebase is needed here again.

**Comment by ubermichael** [comment]
Ok. Everything looks good now on my end after rebasing it to ojs-dev-2_4. 

Is there anything I should be doing to avoid all the rebasing?

**Comment by asmecher** [comment]
Merged and cherry-picked to ojs-stable-2_4_6. I think you're having to rebase because you have multiple PRs that touch the same code; when we merge one, it means the others will need a rebase.

**Pull request merged**